common/src/stack/command/stack/argument_processors/pallet.py

```diff
@@ -2,6 +2,7 @@
 from operator import attrgetter
 import re
 import subprocess
+import stack.commands
```

Comment:
Instead of importing all of `stack.commands`, could you just:
```
from stacki.commands import Log
```

Then change your 5 calls to `Log` so that they are just the bare function call without the namespace.